docs/pages/access-controls/sso/okta.mdx

```diff
@@ -1,107 +1,131 @@
 
-## Configure Okta
+## Step 1/4. Create & assign groups
+
+### Create Groups
+
+Create two groups: "okta-dev" and "okta-admin". In the Okta dashboard go to the
+navigation bar and click **Directory** -> Groups, then **Add group**:
+
+![Create Group Devs](../../../img/sso/okta/okta-saml-2.1@2x.png)
+
+Repeat for the admin group:
+
+![Create Group Devs](../../../img/sso/okta/okta-saml-2.2.png)
+
+## Step 2/4. Configure Okta
 
-First, create a SAML 2.0 Web App in Okta configuration section
+First, create a SAML 2.0 Web App in Okta:
 
-### 1. Select Create New App
+From the main navigation menu, select **Applications** -> Applications, and click
+**Create App Integration**. Select SAML 2.0, then click **Next**.
 
-Select Web Platform and SAML 2.0 Sign On Method.
 ![Create APP](../../../img/sso/okta/okta-saml-1.png)
 
-## Configure the App
+On the next screen (**General Settings**), provide a name and optional logo for your new app, then
+click **Next**. This will bring you to the **Configure SAML** section.
+
+### Configure the App
 
-We are going to map the Okta groups we've created above to the SAML Attribute
-statements (special signed metadata exposed via a SAML XML response).
+Provide the following values to their respective fields:
 
-GENERAL
+#### General
 
-- Single sign on URL `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
-- Audience URI (SP Entity ID)`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+- Single sign on URL: `https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`
+- Audience URI (SP Entity ID): <nobr>`https://<cluster-url>/v1/webapi/saml/acs/new_saml_connector`</nobr>
 - Name ID format `EmailAddress`
 - Application username `Okta username`
 
-SINGLE ATTRIBUTE STATEMENTS
+Replace `<cluster-url>` with your Teleport proxy address or cloud tenant (e.g. `mytenant.teleport.sh`).
+
+#### Attribute Statements
 
 - Name: `username`  | Name format: `Unspecified` | Value: `user.login`
 
-GROUP ATTRIBUTE STATEMENTS
+#### Group Attribute Statements
+
+We will map our Okta groups to the SAML Attribute statements (special signed
+metadata exposed via a SAML XML response).
 
 - Name: `groups` | Name format: `Unspecified`
 - Filter: `Matches regex` |  `.*`
 
-![Configure APP](../../../img/sso/okta/setup-redirection.png)
-
-### Note: RegEx requires `.*`
+The configuration page should now look like this:
 
-![Configure APP](../../../img/sso/okta/regex.png)
+![Configure APP](../../../img/sso/okta/setup-redirection.png)
 
-<Admonition
-  type="tip"
-  title="tip"
->
-  Notice that we have set "NameID" to the email format and mapped the groups with
-  a wildcard regex in the Group Attribute statements. We have also set the "Audience"
-  and SSO URL to the same value.
+<Admonition type="warning" >
+RegEx requires `.*`
 </Admonition>
 
-## Create & Assign Groups
+<Admonition type="tip">
+Notice that we have set "NameID" to the email format and mapped the groups with
```

Comment:
It would be good to have some information as to why the values we mention in this `Admonition` are especially important. Is there a reason we tell the reader about these and not the other values we mention in the instructions above?

Reply:
I've added some explanation here, leaving unresolved for you to 👍 or 👎  it.